web/src/components/Users/index.jsx

```diff
@@ -66,19 +41,30 @@ export default class Users extends React.Component {
       <div style={{ margin: "0 25%" }}>
         <h2>Users</h2>
         <List>
-        {this.state.users.map(
-          (user, index) => 
+          {this.props.users.map(
+            (user, index) =>
               <ListItem key={index} primaryText={user.name}
-              onClick={ ()=>this.props.history.push(`/user/${user.id}`)}
-              rightIconButton={
-                <IconButton onClick = {()=>{this.deleteUser(user.id)}} tooltip="Delite">
-                    <ActionDelete />
-                </IconButton>
-                }/>
+                onClick={() => this.props.history.push(`/users/${user.id}`)}
+              />
           )
-        }
-    </List>
+          }
+        </List>
       </div>
     )
   }
-}
\ No newline at end of file
+}
+
+const mapStateToProps = (state) => {
+  return {
+    fetching: state.loginReducer.fetching,
+    users: state.userReducer.users
+  }
+}
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    loadingUsers: bindActionCreators(UsersAction.loadingUsers, dispatch)
```

Comment:
bindActionCreators should be one layer up

Reply:
`(dispatch) => bindActionCreators({ loadingUsers: UsersAction.loadinUsers }, dispatch)`